### ISSUE TITLE
bug: Google Login doesn't work on safari

### ISSUE THREAD

**Issue by WcaleNieWolny** [Cap-go/capacitor-social-login#36]
If you try to use login via Google on Safari the login will fail.

I tested both macOS Safari and iOS Safair

**Comment by playaz87** [comment]
Also experiencing this.

The first time `SocialLogin.login()` is called on Safari it gives this error:

`[GSI_LOGGER]: Failed to open popup window on url: https://accounts.google.com/o/oauth2/v2/auth?.... Maybe blocked by the browser?`

The second time it works.

**Comment by WcaleNieWolny** [comment]
What version of the plugin do you use?

**Comment by playaz87** [comment]
I'm using https://github.com/Cap-go/capacitor-social-login/pull/57/commits/710cc11359df47992e2664b80f3626cb52fa5b45

**Comment by viktorisacenko** [comment]
I found out that this bug only occurs if you include the initialization process in the login function. Then the popup only opens the second time. However, if you set up the initialization process beforehand (in mounted) in vue 3, for example, then it works as expected.

**Comment by WcaleNieWolny** [comment]
Yes, this is somewhat to be expected. Initialize will load external scripts from Google by creating a "script" tag. This takes time, and I don't think the plugin awaits the scrips to finish loading before returning from initialize.

**Comment by riderx** [comment]
Can we add it to the doc?

**Comment by riderx** [comment]
done
